src/components/learner-credit-management/data/hooks/useOfferRedemptions.js

```diff
@@ -63,14 +63,17 @@ const useOfferRedemptions = (
   const fetchOfferRedemptions = useCallback((args) => {
     const fetch = async () => {
       try {
+        const shouldFetchSubsidyTransactions = budgetId && isTopDownAssignmentEnabled;
         setIsLoading(true);
         const options = {
           page: args.pageIndex + 1, // `DataTable` uses zero-indexed array
           pageSize: args.pageSize,
-          ignoreNullCourseListPrice: true,
         };
+        if (!shouldFetchSubsidyTransactions) {
+          options.ignoreNullCourseListPrice = true;
+        }
         if (budgetId !== null) {
-          options.budgetId = budgetId;
+          options[isTopDownAssignmentEnabled ? 'subsidyAccessPolicyUuid' : 'budgetId'] = budgetId;
```

Comment:
Should this key off shouldFetchSubsidyTransactions instead?
```suggestion
          options[shouldFetchSubsidyTransactions ? 'subsidyAccessPolicyUuid' : 'budgetId'] = budgetId;
```

Reply:
It could; I will update.